documentation/asciidoc/accessories/iqaudio/getting_started.adoc

```diff
@@ -2,24 +2,24 @@
 
 === Creating a toy chatter box
 
-As an example of what Raspberry Pi Audio boards can do, let's walk through the creation of a toy chatter box. Its on-board microphone, programmable button, and speaker driver make the Codec Zero an ideal choice for this application. 
+As an example of what Raspberry Pi Audio Boards can do, let's walk through the creation of a toy chatter box. Its on-board microphone, programmable button and speaker driver make the Codec Zero an ideal choice for this application. 
 
 image::images/Chatter_Box.jpg[width="80%"]
 
-A random pre-recorded 5 second audio clip will be played when the button is pressed. After holding for ten seconds, a notifying burp sound will be emitted, after which a new 5 second clip will be recorded. Lastly, holding for more than 20 seconds will play a second burp sound and then erase all previous recordings.
+A random pre-recorded five-second audio clip will be played when the button is pressed. After holding for ten seconds, a notifying burp sound will be emitted, after which a new five-second clip will be recorded. Holding the button down for more than 20 seconds will play a second burp sound, and then erase all previous recordings.
 
 === Hardware and wiring
 
-For this project, any small passive speaker should be sufficient, we're using one available https://shop.pimoroni.com/products/3-speaker-4-3w?variant=380549926[here], which is a 3" 4Ω - 5W Speaker. We have also used an illuminated momentary push button button and a laser cut box to house all the components, but both are entirely optional. This example will work just using the Codec Zero on-board button, which is pre-wired to GPIO 27. Alternatively, you may use any momentary push button, such as those available https://shop.pimoroni.com/products/mini-arcade-buttons?variant=40377171274[here].
+For this project, any small passive speaker should be sufficient. We're using one available https://shop.pimoroni.com/products/3-speaker-4-3w?variant=380549926[here], which provides 5W of power at 4Ω. We have also used an illuminated momentary push button, and a laser-cut box to house all the components; but both are entirely optional. This example will work just using the Codec Zero's on-board button, which is pre-wired to GPIO 27. (Alternatively, you can use any momentary push button, such as those available https://shop.pimoroni.com/products/mini-arcade-buttons?variant=40377171274[here].)
```

Comment:
Not sure if this is the right wording, as the speaker is only a passive device, and so doesn't **provide** power? Perhaps "which handles 5W of power"? :shrug:

Reply:
Fixed following your suggestion, @lurch; but I'm still not totally happy with the wording (can't think of anything better, though). Any thoughts, @aallan?